pkg/ebpftracer/c/headers/common/buffer.h

```diff
@@ -476,10 +484,11 @@ statfunc int save_args_to_submit_buf(event_data_t *event, args_t *args)
     return arg_num;
 }
 
-#define events_ringbuf_submit(p, id, ret)        do_ringbuf_submit(&events, p, id, ret, true, EVENTS_RINGBUF_DISCARD)
-#define signal_events_ringbuf_submit(p, id, ret) do_ringbuf_submit(&signal_events, p, id, ret, true, SIGNAL_EVENTS_RINGBUF_DISCARD)
+#define events_ringbuf_submit(p, id, ret)        do_ringbuf_submit(&events, p, id, ret, true, 0, EVENTS_RINGBUF_DISCARD)
+#define signal_events_ringbuf_submit(p, id, ret) do_ringbuf_submit(&signal_events, p, id, ret, true, 0, SIGNAL_EVENTS_RINGBUF_DISCARD)
+#define skb_events_ringbuf_submit(p, id, ret)    do_ringbuf_submit(&events, p, id, ret, false, BPF_RB_FORCE_WAKEUP, EVENTS_RINGBUF_DISCARD)
```

Comment:
Forcing wakeup on each event submit will introduce over, as now a epoll notification is send even though no reader is waiting.